packages/teams-js/src/public/sharing.ts

```diff
@@ -194,4 +198,68 @@ export namespace sharing {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.sharing ? true : false;
   }
+
+  /**
+   * Namespace to get the list of content shared in a context
+   *
+   * @beta
+   */
+  export namespace history {
+    /**
+     * Represents the data returned when calling {@link sharing.history.getContent}
+     *
+     * @beta
+     */
+    export interface IContentResponse {
+      /** Id of the app where the content was shared from */
+      appId: string;
+      /** Title of the shared content */
+      title: string;
+      /** Reference of the shared content */
+      contentReference: string;
+      /** Id of the thread where the content was shared. This is a UUID */
+      threadId: string;
+      /** Id of the user who shared the content. This is a UUID */
+      author: string;
+      /** Type of the shared content.
+       * For sharing to Teams stage scenarios, this value would be ShareToStage
+       */
+      contentType: string;
+    }
+
+    /**
+     * Get the list of content shared in a Teams meeting
+     *
+     * @beta
+     */
+    export function getContent(): Promise<IContentResponse[]> {
+      ensureInitialized(runtime, FrameContexts.sidePanel, FrameContexts.meetingStage);
+      if (!isSupported()) {
+        throw errorNotSupportedOnPlatform;
+      }
+
+      return sendAndHandleSdkError(
+        getApiVersionTag(sharingTelemetryVersionNumber_v1, ApiName.Sharing_History_GetContent),
+        'sharing.history.getContent',
+      ).then((contentDetails: IContentResponse[]) => {
+        contentDetails.map((contentDetails) => {
+          validateUuid(contentDetails.author);
```

Comment:
>validateUuid [](http://example.com/codeflow?start=10&length=12)

I looked into the implementation of these functions and they seem mostly correct but they weirdly "allow" for the input parameter to be `undefined` or `null` without complaining. This seems wrong to me (in general, not specifically for your scenario), since `undefined` and `null` are both invalid UUIDs.

To be sure your validation is comprehensive, can I suggest you check here to ensure they aren't `null` or `undefined`? I know it's lame that the shared function that someone wrote doesn't do the right thing and that you can't just use that shared function, but I'd rather you have the correct validation and not suffer because of a suboptimal shared function. You could even help the whole codebase by adding a new function next to `validateUuid` in the shared location that doesn't tolerate `undefined` or `null`, e.g. `validateDefinedUuid` or something. Whatever works for you.

(unless it is actually valid for these two values to be `null` or `undefined` for you, in which case you should update their types on the `IContentResponse` interface)